codebase/superdarn/src.lib/tk/gtable.2.0/src/gtable.c

```diff
@@ -286,7 +286,7 @@ int GridTableAddBeam(struct GridTable *ptr,
     if (bm==NULL) return -1;
 
     /* Inertial velocity correction as a function of radar geodetic latitude [m/s] */
-    velco=(2*PI/86400.0)*6356.779*1000*cos(PI*pos->geolat/180.0);
+    velco=(2*PI/86400.0)*RE*1000*cos(PI*pos->geolat/180.0);
```

Comment:
I'm not sure this change is appropriate - here the velocity correction factor is a function of the equatorial Earth radius and latitude, and therefore should not use a mean Earth radius value.  Although it isn't clear why a polar radius was originally used here.

Reply:
@egthomas, theoretically, shouldn't the radius at the geographic latitude be used?  If so, I believe the mean earth radius is the most appropriate (given our degree of exactness required).

Reply:
@aburrell the `cos(PI*pos->geolat/180.0)` term on the right accounts for the grid cell's actual geographic latitude.  If you scale a mean Earth radius by the cosine of the latitude instead of an equatorial value the calculated velocity will be incorrect.  Here's an example I found:  https://www.nasa.gov/pdf/142825main_Bobsled_Launch.pdf

Reply:
Thanks for the example @egthomas!  In this case, I'd recommend adding the equatorial radius to the constants (`RE_EQ`?).

Reply:
@aburrell No, I think you are correct. Unless I'm misunderstanding something, the point is to calculate the linear velocity at the surface of the Earth at the latitude `geolat` right @egthomas? If this is the case, then to be precise, one needs to use the "radius from the axis of rotation" at `geolat`, which requires knowledge of the radius of the Earth at that latitude (one must correct for the ellipsoidal shape of the Earth). The PDF linked to assumes (implicitly) a spherical Earth:

> This equation quantifies the effect of latitude on initial velocities for any location on Earth, and it works well, **despite the slight oblateness** (being compressed or flattened at the poles) of Earth.

So that means it will "work well" using a mean radius as well. I don't see an elliptical correction being performed anywhere in the source code for `gtable.c`, so picking the polar, equitorial, or mean radius does not, in a meaningful way, make anything more or less precise.

If the point is to be consistent, using the mean radius that is defined is fine. If the point is to be more precise, then we really need to worry about the ellipsoidal correction that is needed.

So, I suggest we shelvethis and defer it to another PR if the we decide we need to be more precise.

Reply:
Just for perspective on how different the velocities would be using the equatorial vs. mean radius, we're talking `(6378.2-6371.2)/6378.2 = 0.1%`. We already had this discussion earlier in #268 regarding the `RPos*` and `RadarPos*` functions and agreed that we would use a common radius.

Reply:
my apologies @mts299 @aburrell @asreimer , I haven't been at my sharpest these last few weeks with the lack of sleep. This inertial velocity term was something I vaguely remember intending to update a few years ago to use the geodetic latitude at the grid cell location rather than a fixed Earth radius value but never got around to doing so. @asreimer is correct that the practical impact on the calculated velocity is very small, so the proposed changes in this pull request are fine as-is.

Reply:
No worries. Hopefully you can get some needed sleep!